packages/core/src/repositories/evaluationResultsRepository/index.ts

```diff
@@ -127,7 +130,23 @@ export class EvaluationResultsRepository extends Repository<EvaluationResultDto>
       )
       .orderBy(desc(evaluationResults.createdAt))
 
-    return Result.ok(result.map(this.parseResult))
+    return Result.ok(results.map(EvaluationResultsRepository.parseResult))
+  }
+
+  async findByDocumentLogIds(documentLogIds: number[]) {
+    const results = await this.scope
+      .where(
+        and(
+          isNull(runErrors.id),
+          isNotNull(evaluationResults.resultableId),
+          isNotNull(evaluationResults.evaluatedProviderLogId),
```

Comment:
What happens with manual logs? They don't have evaluatedProviderLogId. Do they?

Reply:
Manual logs also have an evaluated provider log (at least what I understand from the code).
BTW, These are the same filters as with the original EvaluationsRepository scope. As 'where's cannot be extended I just copied the filters.

Reply:
Ah yes sorry. I miss read. Is evaluated log not evaluation log 👍